src/main/java/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -55,6 +55,8 @@ public CollectorSubsystem(XCANSparkMax.XCANSparkMaxFactory sparkMaxFactory, Prop
         pf.setPrefix(this);
         intakePower = pf.createPersistentProperty("intakePower", 1);
         ejectPower = pf.createPersistentProperty("retractPower", -1);
+        currentIntakeTime = pf.createPersistentProperty("currentIntakeTime", 0);
```

Comment:
Both of these seem like you should use createEphemeralProperty instead. Persistent properties are for settings and get saved on change. Ephemeral properties are for state.